bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/handler/ServerHandler.java

```diff
@@ -179,6 +180,33 @@ private void fetchSystemConfig() throws FreeboxException {
         }
     }
 
+    private void fetchVpnStatus() throws FreeboxException {
+        var vpnManager = getManager(VpnServerManager.class);
+        var servers = vpnManager.getDevices();
+        servers.forEach(vpnServer -> {
+            var groupName = vpnServer.name().replace("_", "-");
+            updateChannelString(groupName, VPN_STATE, vpnServer.state());
+            updateChannelDecimal(groupName, VPN_CONNECTIONS, vpnServer.connectionCount());
+            updateChannelDecimal(groupName, VPN_AUTHENTICATED, vpnServer.authConnectionCount());
```

Comment:
Maybe, you could check that `groupName` is either wireguard or openvpn-bridge or openvpn-routed or pptp before setting channel states.